src/platform/forms-system/src/js/definitions/email.js

```diff
@@ -9,6 +9,8 @@ export default function uiSchema(title = 'Email address') {
     'ui:errorMessages': {
       format:
         'Enter a valid email address using the format email@domain.com. Your email address can only have letters, numbers, the @ symbol and a period, with no spaces.',
+      pattern:
+        'Enter a valid email address using the format email@domain.com. Your email address can only have letters, numbers, the @ symbol and a period, with no spaces.',
```

Comment:
I've seen "Please enter your email address using this format: X@X.com" error message used in other places. Did this message get content review?

Reply:
Just chiming in - that yes, we worked on this error message with @DanielleThierryUSDSVA & @megzehn after receiving feedback from the 508 office that they wanted a more detailed error message.